src/wren_core.c

```diff
@@ -878,6 +878,24 @@ DEF_PRIMITIVE(num_cos)
   RETURN_NUM(cos(AS_NUM(args[0])));
 }
 
+DEF_PRIMITIVE(num_decimal)
+{
+  double value = AS_NUM(args[0]);
+  RETURN_NUM(value - (int32_t)value);
+}
+
+DEF_PRIMITIVE(num_deg)
```

Comment:
In fact I wasn't really sure both `deg` and `rad` are so useful for everybody (due to the fact that they could be easily computed in Wren itself), but due to the presence of the trigonometric methods, I thought it would be funny to write something like
```
270.rad.sin
```
:)